src/components/GameOver.js

```diff
@@ -3,9 +3,10 @@ import React from "react";
 const GameOver = (props) => {
   const { score } = props;
   return (
-    <section id="gameOver">
-      <h4>Good Job!</h4>
-      <p>Your Score is {score}</p>
+    <section id="gameOver" className="pill">
+      {score >= 100 ? <h4>Good Job!</h4> : null}
+      {score >= 50 && score < 100 ? <h4>Pretty Good</h4> : null}
+      {score < 50 ? "Participation trophy" : null}
```

Comment:
Go ahead and create a test file testing the GameOver component that has 3 tests to cover these 3 cases: passing in a different score property, assert that it renders the correct text output